test/binary/binary-boundary.test.py

```diff
@@ -10,14 +10,16 @@
 from internal.memcached_connection import MemcachedBinaryConnection
 from internal.memcached_connection import STATUS, COMMANDS
 
+from internal.compat import string_types
+
 mc = MemcachedBinaryConnection("127.0.0.1", iproto.py_con.port)
 
 def iequal(left, right, level = 1):
     if (left != right):
         tb = traceback.extract_stack()[-(level + 1)]
-        print "Error on line %s:%d: %s not equal %s" % (tb[0], tb[1],
-                repr(left), repr(right))
-        if (isinstance(left, basestring)):
+        print("Error on line %s:%d: %s not equal %s" % (tb[0], tb[1],
+                repr(left), repr(right)))
```

Comment:
To be honest, it would be a bit easier to look over the changes if all those print -> print() ones would be in a separate commit. I already looked over, so it is not important for me to review. Feel free to resolve the comment without changes and save some time.

Reply:
done